unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/TaskbarMoreMenu.cs

```diff
@@ -206,4 +217,35 @@ private void ToggleHideUI()
         CommonScriptableObjects.allUIHidden.Set(!CommonScriptableObjects.allUIHidden.Get());
         view.moreButton.SetToggleState(false);
     }
+    private void ReportBug()
+    {
+        var userProfile = UserProfile.GetOwnUserProfile();
+        var nametag = UnityWebRequest.EscapeURL(userProfile.userName);
+        var kernelConfig = KernelConfig.i.Get();
+        var realm = DataStore.i.playerRealm.Get()?.serverName;
+        var unityVersion = GetVersion(kernelConfig.rendererURL);
+        var kernelVersion = GetVersion(kernelConfig.kernelURL);
+        var systemInfo = $"{SystemInfo.processorType}, " +
+                         $"{SystemInfo.systemMemorySize}MB RAM, " +
+                         $"{SystemInfo.graphicsDeviceName} " +
+                         $"{SystemInfo.graphicsMemorySize}MB";
+        
+        string url = $"{ReportBugURL}" +
+                     $"&unity={unityVersion}" +
+                     $"&kernel={kernelVersion}" +
+                     $"&nametag={nametag}" +
+                     $"&sysinfo={systemInfo}" +
+                     $"&realm={realm}";
+        
+        WebInterface.OpenURL(url);
+    }
+
+    private string GetVersion(string raw)
+    {
+        var match = Regex.Match(raw, VERSION_PATTERN);
+        if (!match.Success)
+            return raw;
+
+        return match.Value;
+    }
```

Comment:
Applying the resposibility principle, I think these 2 methods shouldn't be in the `TaskbarMoreMenu` class. Both methods are very related to "report a bug" and we should have a specific controller for that.

Reply:
You are right, that class was already convoluted